_infra/helm/frontstage/templates/deployment.yaml

```diff
@@ -117,9 +117,19 @@ spec:
                 name: {{ .Chart.Name }}
                 key: email-salt
           - name: EQ_URL
+            {{- if .Values.dns.enabled }}
             value: "{{ .Values.eq.url }}"
+            {{- else }}
+            value: "http://mock-eq.{{ .Values.namespace }}.svc.cluster.local:{{ .Values.dns.wellKnownPort }}/session?token="
+            {{- end }}
           - name: EQ_V3_URL
+            {{- if .Values.dns.enabled }}
             value: "{{ .Values.eq.v3url }}"
+            {{- else }}
+            value: "http://mock-eq.{{ .Values.namespace }}.svc.cluster.local:{{ .Values.dns.wellKnownPort }}/v3/session?token="
```

Comment:
same as above something like `it should be something like `"http://$(MOCK_EQ_HOST):$(MOCK_EQ_SERVICE_PORT)"``